templates/jv_change_register_now_button_text.php

```diff
@@ -0,0 +1,7 @@
+<?php
+// Remove the <?php tag if added to your functions.php
+// It changes the Register Now button string (Located on the Event page)
+add_filter( 'FHEE__EE_Ticket_Selector__display_ticket_selector_submit__btn_text', 'change_register_now_text' );
+function change_register_now_text() {
```

Comment:
Incorrect function name based on our readme guidelines: https://github.com/eventespresso/ee-code-snippet-library#remember-the-following-when-creating-a-pull-request

 ```suggestion
function jv_ee_change_register_now_text() {
```

Note would also need to change the name used in the filter.